torchaudio/functional/functional.py

```diff
@@ -77,7 +77,8 @@ def spectrogram(
         power (float or None): Exponent for the magnitude spectrogram,
             (must be > 0) e.g., 1 for energy, 2 for power, etc.
             If None, then the complex spectrum is returned instead.
-        normalized (bool): Whether to normalize by magnitude after stft
+        normalized (bool or str): Whether to normalize by magnitude after stft. If input is str, choices are
```

Comment:
Can you update the docstrings of corresponding transforms?

Reply:
Also it's better mention if `normalized` is of `bool` type and is `True`, what's the mapped `str` option for it.